build/ci/Dockerfile.unittest

```diff
@@ -22,6 +22,7 @@ WORKDIR $WORKDIR
 COPY Makefile .
 
 RUN go get github.com/onsi/ginkgo/ginkgo@v1.16.4 \
+    && go install sigs.k8s.io/controller-runtime/tools/setup-envtest@v0.0.0-20220322150744-c46b410039fd \
```

Comment:
I think these are being re-downloaded too much on my pc.
could you place them before the `COPY`?

Reply:
done

Reply:
I still have this issue with `kustomize` and `controller-gen` :(
do we really need `go-get-tool` to get them? or could we get them in the dockerfile directly like `ginkgo`/`setup-envtest`?

Reply:
we are using the generated commands, if you think we should do it in another way, open a ticket, I don't know the answer right now